regression/cbmc/union10/union_list2.desc

```diff
@@ -11,8 +11,8 @@ Value sets do not record byte-extract operations with sufficient detail:
   struct list_item *p1 = u.my_list.index;
   struct list_item *p2 = p1->previous;
 yields
-  (29) p1!0@1#2 == byte_extract_little_endian(u!0@1#4, 8l, struct list_item { unsigned int value; unsigned int $pad1; struct list_item *previous; } *)
+  (29) p1!0@1#2 == byte_extract_little_endian(u!0@1#4, 8l, struct list_item \{ unsigned int value; unsigned int $pad1; struct list_item *previous; \} *)
```

Comment:
why not match the `(` braces, too?

Reply:
Ah good catch, I didn't realise that Perl parentheses work like that (in vim regexes it's the other way round)

Reply:
because it's marked `KNOWNBUG` ?